File: Leetcode/154-Find-Minimum-in-Rotated-Sorted-Array-II/FindMinimumInRotatedSortedArrayII.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMin = function(nums) {
    var res = nums[0]
    for(var i=1; i<nums.length; i++) {
        res = Math.min(res, nums[i])
    }
    return res
};

File: Leetcode/154-Find-Minimum-in-Rotated-Sorted-Array-II/FindMinimumInRotatedSortedArrayII.py
class Solution(object):
    def findMin(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        # if not nums or len(nums)==0:
        #     return 0
        res = nums[0]
        for i in nums:
            res = min(res, i)
        return res
    
